crates/egui-wgpu/src/lib.rs

```diff
@@ -106,6 +151,24 @@ impl RenderState {
     }
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+fn describe_adaptors(adaptors: &[wgpu::Adapter]) -> String {
```

Comment:
```suggestion
fn describe_adapters(adapters: &[wgpu::Adapter]) -> String {
```
I like your consistency though :)